CountryPickeriOS13+/CountryPickerSPM/Sources/CountryPickerSPM/Country.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  Country.swift
+//  CountryCodeInSwift3
+//
+//  Created by Cl-macmini-100 on 12/19/16.
+//  Copyright © 2016 Suryakant. All rights reserved.
+//
+
+import Foundation
+import UIKit
```

Comment:
It would be better if we can separate out framework dependencies and define layers for managing business and app logic.

Reply:
uiimage should not need Now I think we can remove this. will check this.